### PR TITLE
playbooks for vmss update demo

### PR DESCRIPTION
**Pull request by zikalino**
## Purpose
Adding new samples to illustrate how to capture VM images and use them in VMSS

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## How to Test
*  Install ansible

```bash
$ pip install ansible[azure]
```

*  Install ansible role

```bash
$ ansible-galaxy install Azure.azure_preview_modules
$ pip install -r ~/.ansible/roles/Azure.azure_preview_modules/files/requirements-azure.txt
```

*  Get the sample playbook

```
git clone https://github.com/Azure-Samples/ansible-playbooks.git
cd ansible-playbooks
git checkout [branch-name]
```

* Test the code
```bash
ansible-playbook 01-create-vms.yml  --extra-vars "resource_group=myrg"
ansible-playbook 02-capture-images.yml --extra-vars "resource_group=myrg"
ansible-playbook 03-create-vmss.yml --extra-vars "resource_group=myrg"
ansible-playbook 04-update-vmss-image.yml --extra-vars "resource_group=myrg"
```

## What to Check
Verify that the all the playbooks were successfully run.

### CODE REVIEW

File: vmss_images/vmss-update.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## Step Four - Update Image Reference in VMSS and Upgrade Instances
+
+Run final playbook to replace image A with image B:
+
+```
+ansible-playbook 04-update-vmss-image.yml --extra-vars "resource_group=myrg"
+```
+
+Now press **F5** in the browser to reload page and see that image gas changed:
```
Comment (yungezz) [inline vmss_images/vmss-update.md]: type `gas`

File: vmss_images/02-capture-images.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Description
+# ===========
+# This playbook creates an Azure VM with public IP, and then dumps it using facts
+
+- name: Create Azure VM
```
Comment (yungezz) [inline vmss_images/02-capture-images.yml]: update name and description above to align with playbook

File: vmss_images/02-capture-images.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  hosts: localhost
+  connection: local
+  roles:
+    - Azure.azure_preview_modules
+  vars:
+    vm_name: vmforimage
```
Comment (yungezz) [inline vmss_images/02-capture-images.yml]: define global var files?

File: vmss_images/03-create-vmss.yml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+- hosts: localhost
+  roles:
+    - Azure.azure_preview_modules
+  vars:
+    vmss_name: vmsstest
+    location: eastus
+    admin_username: vmssadmin
+    admin_password: User123!!!abc
+    vm_name: vmforimage
+    image_name: "image_vmforimageA"
```
Comment (yungezz) [inline vmss_images/03-create-vmss.yml]: again, pls define global var  file to make a product level sample

File: vmss_images/04-update-vmss-image.yml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+      upgrade_policy: Manual
+      tier: Standard
+      managed_disk_type: Standard_LRS
+      os_disk_caching: ReadWrite
+      image:
+        name: "{{ image_name }}"
+        resource_group: "{{ resource_group }}"
+      load_balancer: "{{ vmss_name }}lb"
+
+  - name: List all of the instances
```
Comment (yungezz) [inline vmss_images/04-update-vmss-image.yml]: could you pls add comments this step will be covered by facts module in 2.8

File: vmss_images/02-capture-images.yml (hunk 2 above)
Comment (zikalino) [reply vmss_images/02-capture-images.yml]: yes, I still have a few issues like this to fix :-)

File: vmss_images/vmss-update.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+- change index.html to contain **Image A** and **Image B** respectively
+
+Copy and paste IP addresses of both VMs to the browser:
+
+![Public IP Address](images/vmss-update-vms-ip-addresses.png)
+
+
+You should see two different versions of VM:
+
+![VM A](images/vmss-update-browser-vma.png)
```
Comment (yungezz) [inline vmss_images/vmss-update.md]: it's great to have image in doc. could refine the image a little? it's hard to tell something from ip.

**Comment by Fred-sun** [comment]
@zikalino Could you help confirm the PR ready for review or not? Thanks!

**Comment by Fred-sun** [comment]
@zikalino @yuwzho  Could you help take a look this PR and push for merged? Thanks!

**Comment by Fred-sun** [comment]
@zikalino

**Pull request merged**